CMakeLists.txt

```diff
@@ -249,8 +253,12 @@ if(NOT blue_framework_POPULATED)
 	set_target_properties(UpdateLibsAndDLLsForCurrentBuildConfiguration PROPERTIES FOLDER "BlueFramework/Commands")
 endif()
 
-set (CMAKE_PREFIX_PATH "C:/OSGeo4W64/share/cmake/proj4")
-find_package(PROJ4 REQUIRED)
+
+#set (PROJ_INSTALL_DIR "C:/OSGeo4W64/apps/proj-dev/share/cmake/proj4/proj")
+#set (CMAKE_PREFIX_PATH PROJ_INSTALL_DIR)
+

```

Comment:
Remove the commented out stuff that is a remnant of a failed attempt. We try to keep the code at least at the same level of chaos as it was before and not add to it :)

Reply:
Resolved: 213c48b